lib/ansible/modules/network/nxos/_nxos_mtu.py

```diff
@@ -25,7 +25,10 @@
 module: nxos_mtu
 extends_documentation_fragment: nxos
 version_added: "2.2"
-deprecated: Deprecated in 2.3 use M(nxos_system)'s C(mtu) option.
+deprecated:
+  removed_in: "2.3"
```

Comment:
I still see code in this file.  SHould it be removed_in: "2.7" ?

Reply:
Reviewing these by hand again